internal/app/dogma/watch_cmd.go

```diff
@@ -30,14 +33,59 @@ type watchCommand struct {
 	repo      repositoryRequestInfo
 	jsonPaths []string
 	streaming bool
+	listener  string
```

Comment:
how about `listenerFile`?

Reply:
Thanks, that sounds better. updated 7203e94

Reply:
oops, I forgot to add test change to the commit.
Here's amended fix cadda66.